<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioWidget</class>
 <widget class="QWidget" name="ScenarioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDisplays">
      <attribute name="title">
       <string>Displays</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ScenarioView" name="treeView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSources">
      <attribute name="title">
       <string>Sources</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ScenarioFrameView" name="frameView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFilter">
      <attribute name="title">
       <string>Filter</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="FilterWidget" name="frame_filter" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScenarioView</class>
   <extends>QTreeView</extends>
   <header>steamshovel/ScenarioWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ScenarioFrameView</class>
   <extends>QTreeView</extends>
   <header>steamshovel/ScenarioWidget.h</header>
  </customwidget>
  <customwidget>
   <class>FilterWidget</class>
   <extends>QWidget</extends>
   <header>steamshovel/FilterWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
